examples/vic-gov-au/pages/CustomIcon.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <rpl-icon symbol="icon_circle_arrow_right" color="primary" size="m" />
+</template>
+
+<script>
+import RplIcon from '../../../packages/components/Atoms/Icon/Icon'
+import './../assets/ripple-icon/icon_circle_arrow_right'
```

Comment:
This import here is for icons specific to their website and should not be added in Ripple. Or if they want to override Ripple icon they just register the Icon as the same name in Ripple.

This component is an example of how to implement custom icons and what needs to be refactored if we want to go down this path. 

ping @tim-yao
<img width="766" alt="Screen Shot 2020-08-13 at 2 26 58 pm" src="https://user-images.githubusercontent.com/14160054/90094358-13b6a000-dd71-11ea-8e9d-6831ba43649d.png">

Reply:
The import can also be added inside ripple: {} config when `customIcon:true`. We just need index.js with Icon registration inside ripple-icon directory so we can call it in e.g.
```
<% if (options.customIcon) { %>
  // Add custom icons to library.
  import '../assets/ripple-icon/'
<% } %>
```